apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -126,6 +128,33 @@ const PublishNotes: React.FunctionComponent = () => {
           className={modelCustomizationStyles.updateButton}
         />
       </div>
+      {havePropertiesChanged && !saveInProgress && !isReadOnly && (
+        <Alert
+          text={'You have unsaved changes'}
```

Comment:
Are ya'll localizing strings yet? If so, where would you like me put them?

Reply:
Also curious about this, saw we have a few strings in `common` and a few in `aichat`